modules/test/playwright/package.json

```diff
@@ -10,7 +12,15 @@
 	"license": "ISC",
 	"main": "index.js",
 	"name": "@liferay/playwright",
+	"private": true,
+	"resolutions": {
+		"prettier": "2.1.2"
+	},
 	"scripts": {
+		"checkFormat": "liferay-npm-scripts prettier \"**/*.ts\" --list-different --experimental-standalone",
+		"format": "liferay-npm-scripts prettier \"**/*.ts\" --write --experimental-standalone",
+		"playwright": "playwright",
+		"test": "playwright test"
```

Comment:
Do we also want to add the UI or other modes here? I guess we may be able to run `npm run test -- --ui`

Reply:
I'd say we can invoke those with parameters, since it's documented in Playwright. However it's true that some of them may add discoverability :thinking: 

The "problem" is that (at least I) don't know yet which ones will be the most used...

Also, how would we name the `--ui` script? :thinking:

Reply:
You could just do `npm run playwright:ui` or `npm run test:ui`? And then that way we can add a few more as needed, particularly to help with discoverability.

Pretty much whatever we recommend to use in the docs, I would add to "scripts". Ideally people wouldn't have to go to the docs, but they could end up seeing a certain script and googling what the command is.

It is far easier to google "what is playwright test --ui" vs "how do you make the browser pop up for playwright."

The particular case I am thinking of is someone who doesn't interact with tests frequently but only dips in occasionally.

Reply:
Makes sense. However I will leave it for another PR, bc I want to reflect more about it and I'm going to add a new commit to change the number of workers to this PR, so I don't want to bloat it :grimacing:

Reply:
> reflect more about it

Basically test it and see how things go.